### DISCUSSION TITLE
Changelog auf Github

### DISCUSSION THREAD

**Discussion by gerd2005** [thkl/hap-homematic#393] [General]
Hallo, 

sind die Changelogs der vergangenen und aktuellsten Version auch nirgendwo auf Github ersichtlich?

Mit anderen Worten:
Bspw. Habe ich die neue Version 0.0.59 noch nicht installiert. Kann ich diesen Changelog irgendwo sehen, ohne diese Version zu installieren? 

Viele Grüße

**Comment by jb-home** [comment]
Einfach ins das file CHANGELOG.md schauen ;)

https://github.com/thkl/hap-homematic/blob/master/CHANGELOG.md
